sdk/botservice/Microsoft.Azure.Management.BotService/src/Generated/Operations.cs

```diff
@@ -331,14 +326,13 @@ public async Task<AzureOperationResponse<IPage<OperationEntity>>> ListNextWithHt
             string _responseContent = null;
             if ((int)_statusCode != 200)
             {
-                var ex = new CloudException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
+                var ex = new ErrorException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
```

Comment:
This will result in a breaking change do you have the link to the spec change associated with this?

Reply:
this is code auto generated by Autorest. The version of auto rest used is 2.0.4417